pushbullet

```diff
@@ -7,6 +7,8 @@ API_URL=https://api.pushbullet.com/v2
 PROGDIR="$(cd "$( dirname "$0" )" && pwd )"
 unset QUIET
 
+CURL="curl"
+
```

Comment:
when already defining a variable for curl, I would go for $(which curl) instead of a hardcoded curl.